src/Stratis.Bitcoin.Features.Consensus/PowConsensusValidator.cs

```diff
@@ -57,6 +69,12 @@ public virtual void CheckBlockHeader(ContextInformation context)
             context.NextWorkRequired = context.BlockValidationContext.ChainedBlock.GetWorkRequired(context.Consensus);
         }
 
+        /// <summary>
+        /// Context-dependent validity checks.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <exception cref="ConsensusErrors.BadTransactionNonFinal">Thrown if one or more transactions are not finalized.</exception>
+        /// <exception cref="ConsensusErrors.BadCoinbaseHeight">Thrown if coinbase doesn't start with serialized block height.</exception>
         public virtual void ContextualCheckBlock(ContextInformation context)
         {
             this.logger.LogTrace("()");
```

Comment:
lots of hungarian naming stuff in here, maybe elsewhere as well ... can we get rid of it?